tcpClientServer/include/abstracttcpclientserver.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+class AbstractTcpClientServer
```

Comment:
Ti dirò, questa classe base non mi fa impazzire... Alla fine la stiamo usando come base per classi che vivono in due `main` totalmente diversi: solitamente l'ereditarietà si usa per gestire una collezione di oggetti simili tra loro attraverso puntatori alla classe base (ad esempio un `Vector<QObject*>`), ma in questo caso non ci vedo il vantaggio, a fronte di una maggiore complessità del codice.